docs/building-spark.md

```diff
@@ -79,9 +79,9 @@ Example:
 
     ./build/mvn -Pyarn -Dhadoop.version=3.3.0 -DskipTests clean package
 
-If you want to build with Hadoop 2.x, enable hadoop-2.7 profile:
+If you want to build with Hadoop 2.x, enable hadoop-2 profile:
```

Comment:
maybe use monospace font for `hadoop-2`.

Reply:
Done